### ISSUE TITLE
always is false

### ISSUE THREAD

**Issue by dasuma** [BunHouth/geometry-library#10]
im using your libary but i dont understan , the answer always is false+
<img width="493" alt="image" src="https://user-images.githubusercontent.com/7833341/189027375-202b3b1a-ebb2-43e2-985e-2b3686eb497e.png">
`  let response2 =   PolyUtil.isLocationOnPath(
      {lat: 25.771, lng: -80.19}, // point object {lat, lng}
      [
        // poligon arrays of object {lat, lng}
        {lat: 25.774, lng: -80.19},
        {lat: 18.466, lng: -66.118},
        {lat: 32.321, lng: -64.757}
      ]
    );
    
    console.log(response2) // true  `

i used this version
  "node-geometry-library": "^1.2.6",

**Comment by BunHouth** [comment]
@dasuma  can you provide your code snippet?

**Comment by dasuma** [comment]
`
import { PolyUtil} from "node-geometry-library";


export default class MapsController {
  
  public async getCoverage({ request, response }) {
   

    let response2 =   PolyUtil.isLocationOnPath(
      {lat: 25.771, lng: -80.19}, // point object {lat, lng}
      [
        // poligon arrays of object {lat, lng}
        {lat: 25.774, lng: -80.19},
        {lat: 18.466, lng: -66.118},
        {lat: 32.321, lng: -64.757}
      ]
    );
    
    console.log(response2) // true  


    response.status(200);
    return { 'res': response2 }
  }

}
`

**Comment by BunHouth** [comment]
I'm still not clear about your issue. I'm don't find anything wrong.  Response of PolyUtil.isLocationOnPath is true. Is it false on your code?

```let response =   PolyUtil.isLocationOnPath(
  {lat: 25.771, lng: -80.19}, // point object {lat, lng}
  [
    // poligon arrays of object {lat, lng}
    {lat: 25.774, lng: -80.19},
    {lat: 18.466, lng: -66.118},
    {lat: 32.321, lng: -64.757}
  ]
);

console.log(response) // true  
```

**Comment by dasuma** [comment]
yeah, my response un false, i dont understan why? any problem  with specific version? i have   "node-geometry-library": "^1.2.6",
